build.sbt

```diff
@@ -161,30 +166,37 @@ lazy val nscplugin =
       publishArtifact in (Compile, packageDoc) := false
     )
 
-lazy val sbtplugin =
+lazy val sbtPluginSettings =
+  toolSettings ++
+    publishSettings ++
+    ScriptedPlugin.scriptedSettings ++
+    Seq(
+      sbtPlugin := true,
+      scriptedLaunchOpts ++=
+        Seq("-Xmx1024M",
+            "-XX:MaxPermSize=256M",
+            "-Dplugin.version=" + version.value)
+    )
+
+lazy val sbtNative =
   project
-    .in(file("sbtplugin"))
-    .settings(toolSettings)
-    .settings(publishSettings)
+    .in(file("sbt-native"))
+    .settings(sbtPluginSettings)
     .settings(
-      sbtPlugin := true,
-      // Support for scripted tests
-      ScriptedPlugin.scriptedSettings,
-      scriptedLaunchOpts := {
-        scriptedLaunchOpts.value ++
-          Seq("-Xmx1024M",
-              "-XX:MaxPermSize=256M",
-              "-Dplugin.version=" + version.value)
-      },
+      addSbtPlugin("org.scala-native" % "sbt-cross" % "0.1.0-SNAPSHOT"),
+      moduleName := "sbt-native",
```

Comment:
@densh 
Scala.js uses: sbt-scalajs

I like sbt-native since it's short. There is, however, a possible confusion with [sbt-native-packager](https://github.com/sbt/sbt-native-packager).